syncd/RequestShutdownCommandLineOptions.h

```diff
@@ -14,8 +14,12 @@ namespace syncd
 
         SYNCD_RESTART_TYPE_FAST,
 
+        SYNCD_RESTART_TYPE_EXPRESS,
+
         SYNCD_RESTART_TYPE_PRE_SHUTDOWN,
 
+        SYNCD_RESTART_TYPE_PRE_EXPRESS_SHUTDOWN,
+
```

Comment:
why add this in the middle and not the end of the enu m?

Reply:
Corrected.